techlibs/intel/synth_intel.cc

```diff
@@ -108,6 +110,7 @@ struct SynthIntelPass : public ScriptPass {
 			}
 			if (args[argidx] == "-vqm" && argidx + 1 < args.size()) {
 				vout_file = args[++argidx];
+				log_warning("The Quartus backend has not been tested recently and is likely incompatible with modern versions of Quartus.");
```

Comment:
`log_warning` needs a \n at the end of the string

Reply:
Fixed.